client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -630,7 +739,7 @@
                         $scope.origItem = res;
                         $scope.dirty = false;
                         $scope.item = _.create($scope.origItem);
-                        $scope.not_for_publication_visible = $scope.publish_enabled() && res.marked_for_not_publication === false;
+                        //$scope.not_for_publication_visible = $scope.publish_enabled() && res.marked_for_not_publication === false;
```

Comment:
If this is not needed anymore, it's probably better to remove it completely IMO.